source/blood/src/player.cpp

```diff
@@ -942,6 +947,12 @@ void playerStart(int nPlayer)
     pPlayer->nWaterPal = 0;
     for (int i = 0; i < kMaxPowerUps; i++)
         pPlayer->at202[i] = 0;
```

Comment:
I guess this would be better:
```
    for (int i = 0; i < kMaxPowerUps; i++)
    {
        if (i == 15 || i == 18 || i == 21 || i == 25)
            continue;
        pPlayer->at202[i] = 0;
    }
```

Reply:
I agree.